package.json

```diff
@@ -17,8 +17,8 @@
     "typecheck": "turbo run typecheck",
     "knip": "knip",
     "validate-config": "turbo run validate-config",
-    "merge-checks": "turbo run publint manypkg typesync knip typecheck lint validate-config check-deduped-deps are-the-types-wrong",
-    "check-deduped-deps": "pnpm dedupe --check",
+    "merge-checks": "turbo run publint manypkg typesync knip typecheck lint validate-config are-the-types-wrong",
+    "check-deduped-deps": "pnpm dedupe --check --prefer-offline",
```

Comment:
i don't see this in the options https://pnpm.io/cli/dedupe

Reply:
It's a global option (it's not documented as being global, but it works). It's a tad bit faster.

Reply:
Eh, i don't see the need for this